packages/server/api/src/app/flows/flow.module.ts

```diff
@@ -76,6 +87,13 @@ export const flowModule: FastifyPluginAsyncTypebox = async (app) => {
         };
         socket.emit(WebsocketClientEvent.TEST_STEP_FINISHED, response);
       } catch (err) {
+        sendStepFailureEvent({
+          userId: principal?.id ?? '',
+          projectId: principal?.projectId ?? '',
+          stepName: data.stepName,
```

Comment:
let's add here also 
            flowRunId: flowRun.id,
            flowId: flowRun.flowId,

stepName I think would not be useful. Can we instead add block type and action?

Reply:
ah is step name the user given name?